src/routes/UsersRouter.ts

```diff
@@ -214,13 +214,13 @@ export default class UsersRouter {
 
       this.logger.info("Creating a new planet");
 
-      newPlanet.name = gameConfig.startPlanetName;
+      newPlanet.name = gameConfig.server.startPlanet.name;
```

Comment:
IMO "start" (or "new") would be a nice namespace too, and you could just copy the whole planet object, or use it as prototype ([Object.create](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Object/create))